packages/zoltan2/core/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -3544,11 +3544,6 @@ struct Zoltan2_MJArrayType {
 
   KOKKOS_INLINE_FUNCTION
   Zoltan2_MJArrayType(scalar_t * pSetPtr) : ptr(pSetPtr) {};
-
-  Zoltan2_MJArrayType<scalar_t>& operator=(const volatile Zoltan2_MJArrayType<scalar_t>& zmj) {
-      ptr = zmj.ptr;
-      return *this;
-  }
```

Comment:
We don't need `volatile` overloads for `parallel_reduce` anymore. The default copy assignment operator should be fine here.

Reply:
@masterleinad just to clarify, this change is clean up, or is it also required for sycl compilation?

Reply:
It's both. I can find the error message if required.